code/api/tests/test_followers.py

```diff
@@ -109,13 +108,29 @@ def setUpClass(cls):
     def tearDownClass(cls):
         logging.disable(logging.NOTSET)
 
-    def test_get_followers(self):
+    def test_get_followers_empty(self):
```

Comment:
wait, i think this test and next test have same names. can you pls check

Reply:
Good catch! fixed it now